test/Elastic.Apm.Tests/StackTraceTests.cs

```diff
@@ -42,6 +42,8 @@ public async Task HttpClientStackTrace()
 
 		/// <summary>
 		/// Makes sure that the name of the async method is captured correctly
+		/// Also asserts that the line number is 0 in this case - that is because the fixed method name
+		/// does not match the line number in the state machine, so the agent sends LineNo 0.
```

Comment:
Please see my comment for `LineNo = item?.GetMethod().Name == functionName ? item?.GetFileLineNumber() ?? 0 : 0`

Reply:
I changed this one to test the opposite, which matches the current agent behaviour (https://github.com/elastic/apm-agent-dotnet/pull/253#discussion_r291835766)